install/collecting_ncn_mac_addresses.md

```diff
@@ -72,7 +72,27 @@ For help with either of those, see [LiveCD Setup](bootstrap_livecd_remote_iso.md
     pit# sleep 10
     pit# grep -oP "($mtoken|$stoken|$wtoken)" /etc/dnsmasq.d/statics.conf | sort -u | xargs -t -i ipmitool -I lanplus -U $USERNAME -E -H {} power on
     ```
-4. Now wait for the nodes to netboot. You can follow them with `conman -j ncn-*id*-mgmt` (use `conman -q` to see ). This takes less than 3 minutes, speed depends on how quickly your nodes POST.
+4. Now wait for the nodes to netboot. You can follow them with conman - the `-m` option follows the console output in read-only mode or the `-j` option joins an interactive console session. The available node names were listed in step 2 above.  This takes less than 3 minutes, speed depends on how quickly your nodes POST. To exit the console session, use the command `&.` and to see a conman help screen for all supported escape sequences use `&?`.
```

Comment:
- Remove "Now" from the start of this step.
- Instances of conman should be ConMan.
- Can we make a new step after this to exit the console session, and then remove that line from this block of text?